changelog.md

```diff
@@ -9,6 +9,8 @@ when upgrading from a version of rust-sdl2 to another.
 
 [PR #1407](https://github.com/Rust-SDL2/rust-sdl2/pull/1407) Add new use_ios_framework for linking to SDL2.framework on iOS
 
+[PR #1439](https://github.com/Rust-SDL2/rust-sdl2/pull/1439) **BREAKING CHANGE** Implement custom blend modes
```

Comment:
Should this be marked **Maybe breaking change** by the same criteria as [PR #1156] in v0.35.0?